orchagent/portsorch.cpp

```diff
@@ -4879,6 +4879,12 @@ bool PortsOrch::addBridgePort(Port &port)
         return true;
     }
 
+    if (port.m_rif_id != 0)
+    {
+        SWSS_LOG_NOTICE("Interface %s is still a router port", port.m_alias.c_str());
```

Comment:
Suggest to remove "still" from the log message

Reply:
Done.